index.js

```diff
@@ -0,0 +1,306 @@
+(function(f) {
+
+  'use strict';
+
+  /* istanbul ignore else */
+  if (typeof module === 'object' && typeof module.exports === 'object') {
+    module.exports = f();
+  } else if (typeof define === 'function' && define.amd != null) {
+    define([], f);
+  } else {
+    self.sanctuaryTypeClasses = f();
+  }
+
+}(function() {
+
+  'use strict';
+
+  var STC = {};
+  var instances = STC.instances = {Array: {}, Function: {}};
+
+  var FL = {
+    ap: 'ap',
+    chain: 'chain',
+    concat: 'concat',
+    empty: 'empty',
+    equals: 'equals',
+    extend: 'extend',
+    extract: 'extract',
+    map: 'map',
+    of: 'of',
+    reduce: 'reduce',
+    sequence: 'sequence'
+  };
```

Comment:
Well, it gives control of how to bundle the code to the consumer. It also allows us to be much more flexible in which modules we use (as here). It makes the code cleaner.

[It is trivial](https://wzrd.in) to generate standalone bundles from npm modules, and so you don't even need to be using a bundler to use an npm package on the front-end.

If we wanted to have a bower version available, we could bundle sanctuary-js projects into a `dist/` folder.